### DISCUSSION TITLE
Any plan to support Xiaomi BE10000 (AX10000) (RF01)

### DISCUSSION THREAD

**Discussion by idkwhodatis** [immortalwrt/immortalwrt#1140] [Ideas]
### Describe the bug

no official support for Xiaomi BE10000 (AX10000) (RF01)

### ImmortalWrt version

-

### ImmortalWrt release

-

### ImmortalWrt target/subtarget

-

### Device

Xiaomi BE10000 (AX10000) (RF01)

### Image kind

Official downloaded image

### Additional info

Official website for this model can be found [here](https://www.mi.com/xiaomi-routers/10000)
Spec page in English: [here](https://www.androidpimp.com/wireless-routers/xiaomi-router-10000/)
QSDK [here](https://wiki.codelinaro.org/en/clo/qsdk/overview) may come in handy as this model uses a Qualcomm CPU
a crack to enable ssh can be found [here](https://blog.qust.me/be10000)

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by idkwhodatis** [comment]
willing to contribute in any ways :D

**Comment by GombosTom** [comment]
van egy Xiaomi AX10000 ( RC01 ) router, de tiszta kínai, angol nyelv ötlet, és Globál integráció a Home Mi appba érdekelne. Esetleg oldotta már meg ezt valaki?
